src/typescriptSupport.js

```diff
@@ -57,6 +58,11 @@ export async function getTypeScriptSupport(pkg) {
 /**
  * @param {Array<Package>} pkgs
  */
-export function getTSSupport(pkgs) {
-  return Promise.all(pkgs.map(getTypeScriptSupport));
+export async function getTSSupport(pkgs) {
+  const start = Date.now();
+
+  const all = await Promise.all(pkgs.map(getTypeScriptSupport));
+
+  datadog.timing('changelogs.getTSSupport', Date.now() - start);
```

Comment:
this isn't for changelogs, but addMetaData

Reply:
oops, thanks